### DISCUSSION TITLE
Search / Filter Feature for Categories, Tags and other Screens

### DISCUSSION THREAD

**Discussion by mciantar** [firefly-iii/firefly-iii#8810] [Firefly III] [answered]
The Search Bar on the top right does a good job searching certain things, but when I need to look at transactions for a specific category, I have to browse through a number of pages to find what I want. Would it be a good idea that each screen has a filter / search functionality?

**Comment by JC5** [comment, marked as answer]
Yes, this is already on my list.

**Reply by mciantar** [reply]
Thanks! Is there a ticket I can follow so I know when it has been added?

**Reply by JC5** [reply]
It'll be part of the v2 layout. No specific ticket yet.
